main/src/main/java/org/mobilitydata/gtfsvalidator/notice/StartAndEndTimeOutOfOrderNotice.java

```diff
@@ -19,6 +19,11 @@
 import com.google.common.collect.ImmutableMap;
 import org.mobilitydata.gtfsvalidator.type.GtfsTime;
 
+/**
+ * Two {@code GtfsTime} are out of order
+ *
+ * <p>Severity: {@code SeverityLevel.WARNING}
```

Comment:
Hmmm...seems like this should be an obvious error for frequencies.txt start_time and end_time, but the spec doesn't explicitly say it.

@MobilityData/transit-specs?

Reply:
I agree, it doesn't logically make sense. This can be treated as an ERROR.
Registered in Discrepancies Specification vs Validator https://github.com/MobilityData/gtfs-tasks/issues/87